### DISCUSSION TITLE
Budibase 3.0

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#14608] [Announcements]
Budibase 3.0 is coming very soon. Register for the launch week:
https://budibase.com/lp/3/

**Comment by MarleTangible** [comment]
Any documentation about the upcoming features? Or do you think it would spoil the surprises.

**Reply by saz33m1** [reply]
I absolutely love Budibase! Congratulations on the new release. Thank you for the hard work! My only feedback is, that the cloud offering seems to be slow. Not complaining, its an amazing product for the free tier. I am hoping self-hosting can improve performance dramatically.

**Reply by saz33m1** [reply]
I am happy to report **self-hosted Budibase is lightning fast!** ⚡

**Comment by loureirodnl** [comment]
Hi Budibase team
Thank you for your work and excited to see the new features of Budibase 3.0
I registered 3 times on the landing page (with a microsoft corporate email) but I never received any confirmation link or webinar link. Could you maybe share the invitation link ?
Thanks

**Comment by joebudi** [comment]
We're live buddies:
https://budibase.com/blog/updates/v3/launchweek/day1/

**Comment by andyburgessmd** [comment]
What is the plan for when 3.0 will be available for us to use?

**Reply by joebudi** [reply]
Day 1 features are live now.

**Reply by andyburgessmd** [reply]
@joebudi Do we need to do anything in hosted versions to get it? I'm not seeing anything new and 2.33.14 is still latest version.
Also is this going to break things if we update? Been burnt with updates before!

**Reply by mike12345567** [reply]
Hi @andyburgessmd version 3.0.1 has been released, images here:
1. Single image: https://hub.docker.com/layers/budibase/budibase/3.0.1/images/sha256-12a6b73305dd66bffce90a5fd0f3fd206b0fd6c8fb488d63537554abdca295f7?context=explore
2. Docker-compose images: https://hub.docker.com/layers/budibase/apps/3.0.1/images/sha256-037f74ba3435a0de74561364c53cb9f906bb70613c00ac101bed58376e2d9c35?context=explore and https://hub.docker.com/layers/budibase/worker/3.0.1/images/sha256-3dec1315a7b6c5762f48e906e6b88bce9102ecf5b3901050f966d00d9a0f06a6?context=explore

You can follow the steps to update here: https://docs.budibase.com/docs/updating-budibase
